lib/Form/Util/DateConverter.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace MailPoet\Form\Util;
```

Comment:
I don't understand why is this in the `Form` namespace? It seems like it has nothing to do with forms. Does it?